src/js/utils/background.js

```diff
@@ -137,13 +182,28 @@ export const backgroundStyle = (backgroundArg, theme, textColorArg) => {
     theme,
   );
 
-  if (background.image) {
+  let backgroundImage = normalizeBackgroundImage(background, theme);
```

Comment:
I can't seem to get background image + opacity to work via theme definitions the way it is in https://storybook.grommet.io/?path=/story/layout-box-background--background-box

Not necessarily blocking for this PR given the timeframe we'd like to add this in, but something to add in future.

Reply:
I see the issue, but not an immediately elegant solution. Going to remove supporting BackgroundType from base.d.ts and limit to `string | { dark?: string; light?: string }` for now.